test/loopback-dl.test.js

```diff
@@ -1120,7 +1120,51 @@ describe('Model define with relations configuration', function() {
       keyTo: 'id',
       multiple: false,
       polymorphic: {
-        as: 'imageable',
+        reference: 'imageable',
+        foreignKey: 'imageableId',
+        discriminator: 'imageableType',
+      },
+    });
+    done();
+  });
+
+  it('sets up hasMany polymorphic relation with `{polymorphic: belongsToRelationName}`', function(done) {
+    var ds = new DataSource('memory');
+
+    var Picture = ds.define('Picture', {name: String}, {relations: {
+      imageable: {type: 'belongsTo', polymorphic: true},
```

Comment:
> hmm, sorry I don't get what is the difference between this test
you are right! the code is the same for the two tests, either we merge the two in one or i remove the not-tested relation from each one